tests/Commands/ResultsUpdateAllCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace RecipeScraperTests\Commands;
+
+use InvalidArgumentException;
+use RecipeScraper\Factory;
+use RecipeScraperTests\UsesTestData;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\VarExporter\VarExporter;
+
+class ResultsUpdateAllCommand extends Command
+{
+    use UsesTestData;
+
+    protected function configure()
+    {
+        $this->setName('results:update-all')
+            ->setDescription('Update all result files with content from their corresponding HTML file')
+            ->addArgument(
+                'fields',
+                InputArgument::REQUIRED,
+                'A comma separated list of result fields to update'
```

Comment:
I think I would prefer for `fields` to be an option instead of an argument and set the default value to all available fields.

https://symfony.com/doc/current/console/input.html